README.md

```diff
@@ -116,6 +116,70 @@ $ remme account get-balance \
 }
 ```
 
+### Batch
+
+Get a list of batches — ``remme batch get-list``:
+
+| Arguments | Type   |  Required | Description                                         |
+| :-------: | :----: | :-------: | --------------------------------------------------- |
+| ids       | String |  No       | Identifiers to get a list of batches by.            |
+| start     | String |  No       | Parameter to list batches starting from.            |
+| limit     | Integer|  No       | Parameter to limit amount of batches.               |
```

Comment:
Is `start` an identifier of the `block` to start paging (inclusive) from?

`Parameter to limit amount of batches` > `Number of transactions to return`.

Reply:
1) start - Batch identifier to start listing from.
2) Maximum amount of batches to return.